lightkurve/targetpixelfile.py

```diff
@@ -1740,7 +1762,7 @@ def mission(self):
         """'Kepler' or 'K2'. ('MISSION' header keyword)"""
         return self.get_keyword('MISSION')
 
-    def extract_aperture_photometry(self, aperture_mask='default', centroid_method='moments'):
+    def extract_aperture_photometry(self, aperture_mask='default', flux_mode='default', centroid_method='moments'):
```

Comment:
I recommend making `flux_mode="sum"` the default.  This would make the default behavior explicit, and avoids having two options for the same behavior (i.e. `flux_mode="sum"` and `flux_mode="default"` would do the same).

Reply:
This is outdated -- sorry.

Reply:
Done